### ISSUE TITLE
Tha valide mode argument

### ISSUE THREAD

**Issue by najouamghazli** [aditya-88/ASAP#2]
Hello,

Thank you for this tool, seems interesting. I already installed it correctly on Ubuntu but I have some issues when trying to run the command. I think that I didn't quit understand the component of the command line. Here's what I've run just to test it:
`
ASAP_0.33.py F/R/FR/FE/RE/FRE 15.10F_003_C04_2016-05-19.ab1 15.10R_004_D04_2016-05-19.ab1 ../References/Exon_Only_Seq.fa ../References/Amino_acid_Reference.fa ../References/Reference_Seq.fa`

And here's what I got 
```

Welcome to ASAP! version 0.33!


Checking system for installation of required programs and the commandline arguments.
Total 7 tests to perform.

1. SEQTK seems installed!		PASS
2. SeaView seems installed!		PASS
3. BioPython seems installed!		PASS
4. EMBOSS seems installed.		PASS
5. CLUSTALW2 seems installed.		PASS
6. Command arguments seems correct.	PASS
7. NCBI BLAST+ seems to be intalled.	PASS

System check: PASS
Welcome to Automated Sanger Analysis Pipeline (ASAP!) version 0.33 by Aditya Singh
aditya.onco@gmail.com.
Will utilize all 4 available threads of the system for NCBI BLAST.
'F/FE/FR/FRE/R/RE' is not a valid mode argument.
Make sure you gave the mode argument correct (F/FE/FR/FRE/R/RE)
Please try Again
```
Any explanation or suggestion will be appreciated,

Najoua

**Comment by addityea** [comment]
Hi!
Thank you for reaching out.
The program has correctly mentioned the error which has occurred because of the run mode choice made by you.

You’ve to select only one out of the listed arguments, and it depends on your use case. 
Like in the example above, you’ve to use “FRE,” as you are providing both the forward and reverse sequence, along with the exons file for extraction.

I request you to kindly read the manual for better explanation.

In short:

F/FE/FR/FRE/R/RE: Means either one of it based on your use case.

Hope it helps.

Let me know so that I may close this issue.

Thank you!

**Comment by najouamghazli** [comment]
Thank you, it did work. Now I can clearly understund the syntaxe of the command.

Best
